core-services/prow/02_config/openshift/openstack-resource-controller/_prowconfig.yaml

```diff
@@ -0,0 +1,20 @@
+tide:
+  queries:
+  - includedBranches:
+    - main
+    - master
```

Comment:
I don't know if this is auto-generated or not, but if not do we want to drop `master` here? There's no `master` branch upstream.

Reply:
yeah right it was generated. I'll leave it here and clean later eventually.